src/ResultSetBuilder.php

```diff
@@ -60,14 +59,9 @@ public function buildModelFromIndexAndData(string $indexName, $source)
         return $this->buildModel($indexName, $source, []);
     }
 
-    public function buildModelFromDocument(Document|ElasticaDocument $document)
+    public function buildModelFromDocument(Document $document)
```

Comment:
We must keep that to preserve the BC

Reply:
I renamed the imported class. This is BC since I'm supporting `ElasticaDocument`.